### PR TITLE
Chore/neo rpc:  add CLI option to toggle ping first behavor of getNodesBy

### PR DESCRIPTION
**Pull request by phetter**
Now, all methods except "all" can have ping first behavior toggled with -p --ping

### CODE REVIEW

File: src/nodejs/neo-rpc/v2.9.0/client/module/getNodesBy.js [hunk 1]
```diff
@@ -212,7 +227,19 @@ exports.connection = (options) => {
...
+      })
+      .catch (error => {
+        print(__filename + ': getNodesBy.connections()/getNodesBy.ping().error: ' + error)
+      })
+    }
+    else {
+        connectionCallback(options.nodes)
+    }
+
+    function connectionCallback(nodes) {
```
Comment (DalderupMaurice) [inline src/nodejs/neo-rpc/v2.9.0/client/module/getNodesBy.js]: can't you reuse the `tallestCallback`?

File: src/nodejs/neo-rpc/v2.9.0/client/module/getNodesBy.js [hunk 2]
```diff
@@ -267,7 +292,19 @@ exports.version = (options) => {
...
+      })
+      .catch (error => {
+        print(__filename + ': getNodesBy.version()/getNodesBy.ping().error: ' + error)
+      })
+    }
+    else {
+      versionCallback(options.nodes)
+    }
+
+    function versionCallback(nodes) {
```
Comment (DalderupMaurice) [inline src/nodejs/neo-rpc/v2.9.0/client/module/getNodesBy.js]: same here

File: src/nodejs/neo-rpc/v2.9.0/client/module/getNodesBy.js (hunk 1 above)
Comment (phetter) [reply src/nodejs/neo-rpc/v2.9.0/client/module/getNodesBy.js]: Nah, these will eventually be reused from outside of their API expression too. They need to be unique for extensibility, for now.

File: src/nodejs/neo-rpc/v2.9.0/client/module/getNodesBy.js (hunk 2 above)
Comment (phetter) [reply src/nodejs/neo-rpc/v2.9.0/client/module/getNodesBy.js]: See above.

**Pull request merged**